packages/ui-components/src/components/Button/Button.test.js

```diff
@@ -131,4 +131,10 @@ describe("Button", () => {
     expect(screen.getByRole("button")).toHaveAttribute("id", "button-1")
     expect(screen.getByRole("button")).toHaveAttribute("data-lolol")
   })
+
+  test("renders an icon with the defined button title", async () => {
```

Comment:
Please remove async when you do not use await, lint going to throw error later on when we transfer it to TypeScript

Reply:
No problem. I was just following the given test structure, so be aware that most functions are async while not using an await. You might have to fix this later for the remaining ones.